encoders/nlp/TFIDFTextEncoder/model/build_tfidf.py

```diff
@@ -0,0 +1,23 @@
+from sklearn.feature_extraction.text import TfidfVectorizer
+import csv
```

Comment:
this script is not needed?

Reply:
It's needed to genereta the dummy pickle that we can include for tests, just as a helper

Reply:
one of the 2 seems innecessary